src/scrabble.js

```diff
@@ -1,5 +1,82 @@
-function scrabble() {
-
+function scrabble(str) {
+  let counter = 0 // must be local to restart and redefine back to 0 when finished otherwise global scope will add it up
+  if (!str || str.length === 0) {
```

Comment:
Very nice way of verifying if the string is empty! Also I think it could also work with just simply (!str)